pyocd/probe/pydapaccess/dap_access_cmsis_dap.py

```diff
@@ -602,6 +602,8 @@ def _read_protocol_version(self):
         # based on unclear documentation.
         elif (self._vidpid == ARM_DAPLINK_ID) and (fw_version_str in ("0254", "0255", "0256")):
             self._cmsis_dap_version = CMSISDAPVersion.V2_0_0
+        elif (self._vidpid == MICROCHIP_EDBG) and (fw_version_str == "03.25.01B6"):
```

Comment:
Does this version string change with different EDBG firmware releases? It might be necessary to parse the version and use a more complex matching expression.